token-lending/program/tests/helpers/mod.rs

```diff
@@ -300,6 +250,13 @@ pub fn add_reserve(
 
     let current_slot = slots_elapsed + 1;
 
+    let market_price = oracle
+        .price
+        .checked_div(oracle.decimals)
+        .unwrap()
+        .checked_mul(10u64.pow(6))
+        .unwrap();
```

Comment:
Need to update this if price decimal normalization changes.

Reply:
Constant would be nice

Reply:
This has been removed in favor of using `Decimal` for market price and using the price + exponent directly from the oracle.